src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol

```diff
@@ -342,8 +343,19 @@ abstract contract BondingCurveBase_v1 is IBondingCurveBase_v1, Module_v1 {
         if ((_protocolFee + _workflowFee) > BPS) {
             revert Module__BondingCurveBase__FeeAmountToHigh();
         }
-        protocolFeeAmount = _totalAmount * _protocolFee / BPS;
-        workflowFeeAmount = _totalAmount * _workflowFee / BPS;
+        protocolFeeAmount =
+            _protocolFee > 0 ? _totalAmount * _protocolFee / BPS : 0;
```

Comment:
I assume this is for gas saving? I'm wondering if it isn't more efficient to wrap everything into if(protocolFee > 0 ) brackets? Just a thouhgt though, the logic looks right

Reply:
So it was not for gas saving but to set the fee amounts to 0 in case fee percentage is not set. However what you describe is more readable and I didn't think about the fact that the variables are initiated with 0 anyways, making the ternary operators not needed.